netket/experimental/driver/tdvp.py

```diff
@@ -121,7 +122,7 @@ def __init__(
         self.linear_solver_restart = linear_solver_restart
 
         super().__init__(
-            operator, variational_state, integrator, t0=t0, error_norm=error_norm
+            operator, variational_state, solver, integrator, t0=t0, error_norm=error_norm
```

Comment:
```suggestion
            operator, variational_state, solver, t0=t0, error_norm=error_norm, integrator=integrator
```

makes it easy to remove it later